lib/wootric/index.js

```diff
@@ -0,0 +1,146 @@
+
+/**
+ * Initialize Wootric.
+ *
+ * @param {Facade} page
+ */
+
+Wootric.prototype.initialize = function(page){
+  // TODO: fill in the logic that needs to be run for your integration to be
+  // properly initialized. Most often that will include a call to `this.load()`
+  // which comes next.
+  //
+  // Here's what a normal `initialize` method might look like:
+  //
+  //   window.__integration = {};
+  //   window.__integration.apiKey = this.options.apiKey;
+  //   this.load(this.ready);
+  //
+  // Once the integration is ready to receive calls to `identify()`, `track()` etc..
+  // you must invoke `this.ready()`.
+  // for example if the integration is ready once it's loaded, you can call `this.load(this.ready)`
+  // if it's ready immediately after `initialize()` is called you can call `this.ready()` here.
+  window._wootric = this;
+  var self = this;
+  window.wootricSettings = {
+    response_callback: this.trackSurveyResponse,
+    decline_callback: this.trackSurveyDecline,
+    survey_callback: this.trackSurvey
+  };
+
+  window.wootricSettings.account_token = this.options.account_token;
+  //Load our library, once it is loaded, we are ready
+  this.load('library', function(){
+    window.wootric("run");
+    self.ready();
+  });
+};
+
+// Track the survey result.  This is called back once the result is sent to Wootric
+// by way of the wootricSettings.response_callback callback
+Wootric.prototype.trackSurveyResponse = function(score, text){
+  this.track('NPS Response', {
```

Comment:
this doesn't actually send a track event to other tools, that would have to be `this.analytics.track`. At this time, however, we're not accepting the ability to send this information to other tools. We will be supporting this in the coming months. For the time being I mute this functionality.